gradle/libs.versions.toml

```diff
@@ -1,21 +1,21 @@
 [versions]
 # ArcGIS Maps SDK for Kotlin version
-arcgisMapsKotlinVersion = "200.4.0-4189"
+arcgisMapsKotlinVersion = "200.4.0"
 # ArcGIS Maps SDK for Kotlin Toolkit version
-arcgisToolkitVersion = "200.4.0-4189"
+arcgisToolkitVersion = "200.4.0"
```

Comment:
I am hesitant to merge these changes into `v.next` as of now, or at least not until the day of release. Although I would like to merge the lint code improvements to `v.next`, removing the build tags and olympus here will break daily builds. 

Instead, we can prep a `release/200.4.0` branch with these final release changes which will eventually merge to `main`, like we have in previous releases. ([200.2](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/pull/131),[200.3](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/pull/141)). This way we can keep building the SV and update the release branch with any `v.next` improvments until day of release. 

For now, could you add in the build tag and olympus code in this PR to continue with PRs the lint code enhancements?

Reply:
Makes sense. Will add them back. Should I create a new release  branch now, or we can wait until the release note is ready?

Reply:
Just saw your second comment about release branch, nvm:)